keyboards/th96/keymaps/nikchi/config.h

```diff
@@ -0,0 +1,4 @@
+#define TAPPING_TERM 200
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Also, needs a `#pragma once`